src/components/dialog/dialog.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @function UIDialog
+ * @version 0.0.2
+ * @desc display a Dialog
+ */
+
+export default function UIDialog(userOptions) {
+
+    var DOM,
+        state = {};
+        
+    const defaults = {
+        dialog:         '.js-dialog',
+        openBtn:        '.js-dialog-btn',
+        closeBtn:       '.js-dialog-close-btn',
+        backdrop:       '.js-dialog-backdrop',
+        isModal:        false,
+        showBackdrop:   true
+    };
+
+    // Combine defaults with passed in options
+    const settings = Object.assign(defaults, userOptions);
+    
+    init();
+
+    /**
+    * @function init
+    * @desc Initialises the dialog
+    */ 
+    function init() {
+        
+        // Save all DOM queries for future use
+        DOM = {
+            'page': document.querySelectorAll('body')[0],
+            'dialog': document.querySelectorAll(settings.dialog)[0],
+            'openBtn': document.querySelectorAll(settings.openBtn)[0],
+            'closeBtn': document.querySelectorAll(settings.closeBtn)[0],
+            'backdrop': document.querySelectorAll(settings.backdrop)[0]
+        };
+        
+        // Check if gallery exists, return if not
+        if (DOM.dialog === undefined) { return false; }
```

Comment:
I'll also go with the majority.

My personal preference is towards single-line for single statements, for both js and css.

For example, when a map function only performs one operation:

```[1, 2, 3].map(function (num) { return num * 2 })```

but with a second operation comes wrappage:

```
[1, 2, 3, 4].map(num => {
    let multiplier = 2 + num
    return num * multiplier
})
```
Anyway, this is one of those cases where there's no real rational behind it other than I think it looks nicer, so let democracy decide!
 
Another one for the style guide.